batch_seqr_loader/batch_workflow.py

```diff
@@ -90,8 +90,9 @@
     '--seqr-dataset',
     'dataset_name',
     required=True,
-    help='Name of the seqr callset. If a genomics DB already exists for this dataset, '
-    'it will be extended with the new samples',
+    help='Name of the seqr dataset. If a genomics DB already exists for this dataset, '
```

Comment:
Oh, so you can actually add to GenomicsDB after all? Would that cause issues if we ever wanted to remove / rename a sample?

Reply:
```suggestion
    help='Name of the seqr dataset. If a GenomicsDB already exists for this dataset, '
```

Reply:
You can add, but you need to download it to the instance and upload back :(
It will be tricky with removing though. Perhaps would need to re-create a new DB.